docs/README.md

```diff
@@ -128,6 +128,23 @@ cleanlab-docs
 
 7. Navigate to the URL where your GitHub Pages site is published in step 3. The default URL should have the format *https://repository_owner.github.io/cleanlab-docs/*.
 
+# Manually adding build artifacts to the `cleanlab-docs` repo
+
+If you've set up the docs remotely on GitHub, GitHub Actions will automatically build and deploy the docs' build artifacts when [triggered](#cicd-for-cleanlab-docs). If you delete and recreate a release tag, the docs for this tag will be rebuilt and redeployed. On rare occasions, you may want to update your docs without changing your release tag, for example, when you want to fix a typo in the docs, but you've deployed your tag to PyPI or Conda. This can be done by manually adding the docs' build artifacts to the `cleanlab-docs` repo:
+
+1. Create a new branch, make the necessary code changes and push it to your `cleanlab` repo. This automatically triggers GitHub Actions to build the docs artifacts.
+
+2. Open your `cleanlab` repo in your browser, navigate to "Actions", under "Workflow", click "GitHub Pages", then select the workflow that was triggered by the previous step.
+
+3. Ensure that the workflow is completed.
+
+4. Scroll to the bottom, under "Artifacts", and click the "docs-html" to download the docs' build artifacts.
+
+5. Extract the "docs-html.zip" file.
+
+6. Select and upload the updated files to your `cleanlab-docs` repo.
+
```

Comment:
Add comment at the end about what will happen if:

The same release, say it's v2.0.0 is re-tagged after I've replaced some of the files. (I assume all your files in cleanlab-docs/v2.0.0 will be overwritten).

Reply:
Also add the following caveat at the end:

WARNING: Any manual edit added to cleanlab-docs/ that does not live in master branch of cleanlab/ repo will be lost in future cleanlab-versions' docs.  So any edit you want to make in the v2.0.0 docs that you also want to have in the v2.0.1, v2.0.2,... docs needs to also be introduced as a PR to the cleanlab/ repo as well.

Reply:
have added caveats at the start and end of section